### DISCUSSION TITLE
Graylog and Postal

### DISCUSSION THREAD

**Discussion by uhlhosting** [postalserver/postal#1735] [Help with using Postal]
Can someone explain me what integrations exactly are supported on Postal side to create required Inputs in Graylog.

**Comment by willpower232** [comment]
I believe you only need to fill out the configuration and logs should be sent to Graylog.

https://github.com/postalserver/postal/blob/ea10ae18f10ec243c9e30a540a3de4d853155838/config/postal.defaults.yml#L39-L41

**Reply by uhlhosting** [reply]
Any idea what Inputs are used in Graylog for Postal?

**Reply by willpower232** [reply]
I'm afraid I don't know anything about how Graylog is configured.

**Reply by uhlhosting** [reply]
No probs, maybe someone who configured will step forward to explain what Inputs were used.
